API/api/services/event_service.py

```
@@ -1,13 +1,42 @@
     def __init__(self):
         self.repo = EventRepository()
 
+
+    def update_event_date(self, event_id: int, event_date_raw: str):
+        if not event_date_raw:
+            raise ValidationError({"detail": "event_date est requis (YYYY-MM-DD)."})
+
+        try:
+            new_event_date = datetime.fromisoformat(event_date_raw).date()
+        except ValueError:
+            raise ValidationError({"detail": "event_date invalide (format attendu: YYYY-MM-DD)."})
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)